modules/friendlytag.js

```diff
@@ -192,8 +235,12 @@ Twinkle.tag.callback = function friendlytagCallback() {
 						type: 'checkbox',
 						name: 'tags',
 						list: subgroup.map(function (item) {
-							return { value: item.tag, label: '{{' + item.tag + '}}: ' + item.description, subgroup: item.subgroup };
-						})
+							returnValue = undefined;
+							if (redirectTagsPresent.indexOf(item.tag) === -1) {
+								returnValue = { value: item.tag, label: '{{' + item.tag + '}}: ' + item.description, subgroup: item.subgroup };
+							}
+							return returnValue;
+						}).filter(removeUndefined => removeUndefined !== undefined)
```

Comment:
No ES6 means no arrow functions allowed
```suggestion
						}).filter(function(removeUndefined) {
							return removeUndefined !== undefined;
```